benchmarks/local/bench_base.rb

```diff
@@ -0,0 +1,229 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+module TestPuma
+  class BenchBase
+
+    # Signal used to trigger worker GC
+    GC_SIGNAL = 'SIGALRM'
+
+    # We're running under GitHub Actions
+    IS_GHA = ENV['GITHUB_ACTIONS'] == 'true'
+
+    WRK_PERCENTILE = [0.50, 0.75, 0.9, 0.99, 1.0].freeze
+
+    def initialize
+      sleep 5 # wait for server to boot
+
+      @thread_loops       = nil
+      @clients_per_thread = nil
+      @req_per_client     = nil
+      @body_kb            = 10
+      @dly_app            = nil
+      @bind_type          = :tcp
+
+      @ios_to_close = []
+
+      setup_options
+
+      unless File.exist? @state_file
+        puts "can't fined state file '#{@state_file}'"
```

Comment:
I noticed a small typo here! 🙈 

```suggestion
        puts "can't find state file '#{@state_file}'"
```